docs/topics/Gateway.md

```diff
@@ -336,21 +333,34 @@ Bots in under 100 guilds can enable these intents in the bot tab of the develope
 
 ### Privileged Intents
 
-> warn
-> `MESSAGE_CONTENT` will become a privileged intent in Aug 2022. [Learn more here](https://support-dev.discord.com/hc/en-us/articles/4404772028055) or read the guide on [upgrading to commands](#DOCS_TUTORIALS_UPGRADING_TO_APPLICATION_COMMANDS).
-
-Some intents are defined as "Privileged" due to the sensitive nature of the data. Those intents are:
+Some intents are defined as "Privileged" due to the sensitive nature of the data. Currently, those intents include:
 
 - `GUILD_PRESENCES`
 - `GUILD_MEMBERS`
+- [`MESSAGE_CONTENT`](#DOCS_TOPICS_GATEWAY/message-content-intent)
+
+Apps that qualify for verification must be approved for the privileged intent(s) [during verification](https://support.discord.com/hc/en-us/articles/360040720412-Bot-Verification-and-Data-Whitelisting) before they can use them. If your app is already verified and you need to request additional privileged intents, [contact support](https://dis.gd/support).
```

Comment:
Intents must be requested through the dev portal, not through support.

```suggestion
Apps that qualify for verification must be approved for the privileged intent(s) [during verification](https://support.discord.com/hc/en-us/articles/360040720412-Bot-Verification-and-Data-Whitelisting) before they can use them. If your app is already verified, you can request additional privileged intents in the Developer Portal.
```

Also I think the verification team now wants intents to be requested after verification instead of during verification; apps that request intents during verification are often asked to re-apply for intents after verification. that should be clarified and maybe updated here